test/TestAsset/Application.php

```
@@ -49,6 +52,7 @@ public function getEventManager()
      *
      * @return RequestInterface
      */
+    #[Override]
```

PHPStan (2.2.7)

[error] Method LaminasTest\ApiTools\Versioning\TestAsset\Application::getRequest() should return Laminas\Stdlib\RequestInterface but return statement is missing.


(return.missing)